apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -118,17 +118,12 @@ do_send(Socket, Transport, B, M, T, C, MinRTT) ->
         Cutoff when Cutoff > T ->
             do_send_opt(Socket, Transport, B, M, T, C, []);
         _ ->
-            do_send_opt(Socket, Transport, B, M, T, C, [no_rr])
+            do_send_opt(Socket, Transport, B, M, T, C, [{rr, off}])
```

Comment:
The cutoff needs to be handled for the Opts version too.

I would say cutoffs are only taken into account when the `{rr, default}` option is choosen, otherwise the clients wishes have priority?